multigeiger/multigeiger.ino

```diff
@@ -265,6 +261,12 @@ void loop() {
   loop_duration = millis() - current_ms;
   iotWebConf.delay((loop_duration < LOOP_DURATION) ? (LOOP_DURATION - loop_duration) : 0);
 
+  static bool wifi_connected = false;
   wifi_connected = (iotWebConf.getState() == IOTWEBCONF_STATE_ONLINE);
+  status[STATUS_WIFI] = wifi_connected ? ST_WIFI_CONNECTED : ST_WIFI_CONNECTING;
+
+  status[STATUS_BT] = ST_NODISPLAY;  // TODO
+  status[STATUS_LORA] = ST_NODISPLAY;  // TODO
+  status[STATUS_HV] = ST_NODISPLAY;  // TODO
```

Comment:
Hmm, can't we have a more encapsulated way, like e.g. a function in display which is exposed, allowing to switch the status? I'm not too keen on all the includes and modifying variables in other modules... Would also be easier with fixed status numbers

Reply:
could be done, then status would not need to be global any more.

Reply:
done, will update PR soon.